### ISSUE TITLE
Make Glathida functions work in a `pmap`

### ISSUE THREAD

**Issue by JordiBolibar** [ODINN-SciML/ODINN.jl#71]
Right now, calling `get_glathida` in a `pmap` results in:

```julia
ERROR: LoadError: PyError ($(Expr(:escape, :(ccall(#= /home/jovyan/.julia/packages/PyCall/ygXW2/src/pyfncall.jl:43 =# @pysym(:PyObject_Call), PyPtr, (PyPtr, PyPtr, PyPtr), o, pyargsptr, kw))))) <class 'AttributeError'>
AttributeError("Can't pickle local object 'WeakValueDictionary.__init__.<locals>.remove'")
```

Though not essential, since it is reasonably fast with a serial execution, it would be nice to fix this and make it work in parallel.

**Comment by JordiBolibar** [comment]
This is no longer relevant, since now these are downloaded directly in Python using an entity task from OGGM.
